DSCResources/MSFT_xCertReq/MSFT_xCertReq.psm1

```diff
@@ -128,7 +128,7 @@ function Get-TargetResource
     )
 
     # The certificate authority, accessible on the local area network
-    $ca = "'$CAServerFQDN\$CARootName'"
+    $ca = "$CAServerFQDN\$CARootName"
```

Comment:
Change looks good. Please document your change on  the README.md file https://github.com/PowerShell/xCertificate/blob/dev/ReadMe.md